.github/workflows/ci-rsc.yml

```
@@ -0,0 +1,61 @@
+name: ci-rsc
+"on":
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - "packages/plugin-rsc/**"
+      - "pnpm-lock.yaml"
+      - ".github/workflows/ci-rsc.yml"
+
+concurrency:
+  group: ci-${{ github.event.pull_request.number || github.ref }}
```

```suggestion
  group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
```
To align with other workflows. Also `ci-` should probably be `ci-rsc-`.